### ISSUE TITLE
API keys are exported in pak files by default

### ISSUE THREAD

**Issue by sturcotte06** [gridly-spreadsheet-CMS/gridly-ue4-plugin#6]
`UGridlyGameSettings` is defined in `DefaultGame.ini` instead of `DefaultEditor.ini` and will be exported to pak files by default, effectively leaking secrets to users. I think the minimal for a cooked game should be kept in `Gridly` and move everything that isn't related to gameplay to `GridlyEditor`. Since no settings in `UGridlyGameSettings` seems to be gameplay related, it should be moved to `GridlyEditor` as well and should be changed from `Config = Game` to `Config = Editor` so that's it's defined in `DefaultEditor.ini` and excluded from cooked builds.

**Comment by yuhe00** [comment]
The reason for this is that the plugin supports runtime update from Gridly for translators in cooked builds. There are separate API key settings for import and export. The export API key should be private and is only saved locally, but the import one is packaged with the game by default. This should be made more clear though and perhaps solved in a different way.

**Comment by yuhe00** [comment]
Simply requiring the API key to be input to the BP function that downloads and updates translations is perhaps a good solution?

**Comment by yuhe00** [comment]
If it's a BP function, that input could come from any source, eg. an editable text field in the UI so it's not required bundle key with cooked project in any way.

According to Gridly webapp/backend team, It is recommended to create separate users with different levels of access. API keys for these will be different.

**Comment by sturcotte06** [comment]
> If it's a BP function, that input could come from any source, eg. an editable text field in the UI so it's not required bundle key with cooked project in any way.
> 
> According to Gridly webapp/backend team, it is recommended to create separate users with different levels of access. API keys for these will be different.
> 
> EDIT: Actually, it seems like they have added support for creating API keys with restricted access now as well.

Yes but then the player would have to input an API key? Either way, live updates will require an API key being included in the build somehow, or known by players.

**Comment by yuhe00** [comment]
This feature was not intended for live updates for players, but for external translators/translation teams that do not have access to the Unreal project.
